src/lib/components/Picker.react.js

```diff
@@ -9,18 +9,20 @@ import {
 } from '@material-ui/pickers';
 
 type Props = {
-  /** Format to be used in displaying date. Some possibilities:
-   * yyyy-MM-dd
+  /** Format to be used in displaying date. The slashes between values are important, because they tell JavaScript
```

Comment:
Given this change, will this force the formatting (ex: we always have to use yyyy/MM/dd versus opening up the api to accept the formats?)

Reply:
This will not force the formatting if a user decides to supply a different structure. In my tests, even using the previous format of `yyyy-MM-dd`, it does not seem to have the problem it did before.